layouts/404.html

```diff
@@ -0,0 +1,57 @@
+{{- define "main" -}}
+
+<!-- redirection to pkg.go.dev documentation -->
+<script type="text/javascript">
+  let base = window.location.protocol + "//" + window.location.host;
+
+  trimTrailingSlash = function(str) {
+    return str.replace(/\/+$/, "")
+  }
+
+  window.onload = function() {
+    const prefixes = [
+      "/v1/gonum/",
+      "/v1/plot/",
+      "/v1/netlib/",
+      "/v1/hdf5/",
+      "/v1/tools/",
+      "/v1/exp/"
+    ];
+    prefixes.forEach(prefix => {
+      if (window.location.href.startsWith(base+prefix) || window.location.href == trimTrailingSlash(base+prefix)) {
+          let docURL = window.location.href.replace(base, "https://pkg.go.dev/gonum.org")
```

Comment:
perhaps keep the protocol intact?
```suggestion
          let docURL = window.location.href.replace(base, window.location.protocol + "//pkg.go.dev/gonum.org")
```

Reply:
www.gonum.org always redirects to https, but OK.